### PR TITLE
Feature: long literal integration test + fix for the issue

### PR DESCRIPTION
**Pull request by pfrischmuth**
Integration test and fix for #108.

@white-gecko Please review and merge afterwards...

### CODE REVIEW

File: tests/integration/Erfurt/StoreIntegrationTest.php [hunk 1]
```diff
@@ -360,7 +360,63 @@ public function testImportDBPediaResourceMachineLearningAndQueryWithTitleHelperQ
...
+        $this->authenticateDbUser();
+
+        $modelUri = 'http://example.org/updateTest/';
+        $store = Erfurt_App::getInstance()->getStore();
+        try {
+            $store->deleteModel($modelUri);
+        } catch (Exception $e) {/*ignore deletion failure*/}
+        $model = $store->getNewModel($modelUri);
+        $literal= <<<'EOT'
+"""Gerade eben habe ich einen kleinen PHP Skript geschrieben, der sich selbst als Bild einbindet. Das klingt etwas seltsam aber veranschaulicht das Prinzip der <a href=\"http://de.wikipedia.org/wiki/Content_Negotiation\"><em>Content Negotiation</em></a> sehr gut.
```
Comment (k00ni) [inline tests/integration/Erfurt/StoreIntegrationTest.php]: Is it possible to switch to example texts in English instead of German for future work?

Comment (pfrischmuth) [inline tests/integration/Erfurt/StoreIntegrationTest.php]: It is just a copy and pasted snippet of a literal that produced the error... no need to read it

Comment (white-gecko) [inline tests/integration/Erfurt/StoreIntegrationTest.php]: We can even use Hebrew, Chinese or something else for example text I think.

Comment (pfrischmuth) [inline tests/integration/Erfurt/StoreIntegrationTest.php]: when it makes sense yes ;-)

**Comment by pfrischmuth** [comment]
@white-gecko Did you manage to check it?

**Comment by white-gecko** [comment]
I've checked it and the `Erfurt_UtilsTest::testBuildLiteralString`

Failed asserting that two strings are equal.

```
--- Expected
+++ Actual
@@ @@
 '"""This
 is
 a
 multi
 line
-text with\\backslash"""^^<http://www.w3.org/2001/XMLSchema#string>'
+text with\backslash"""^^<http://www.w3.org/2001/XMLSchema#string>'
```

`ontowiki/vendor/aksw/erfurt/tests/unit/Erfurt/UtilsTest.php:77`

**Comment by pfrischmuth** [comment]
You are right... I will fix it...

**Pull request closed without merging**